ChildProject/pipelines/derivations.py

```diff
@@ -5,13 +5,22 @@
 def acoustics():
     pass
 
+
+INTERACTIONS = {
+    'CHI': {'FEM', 'MAL', 'OCH', 'CHI'},
+    'FEM': {'FEM', 'MAL', 'OCH', 'CHI'},
+    'MAL': {'FEM', 'MAL', 'OCH', 'CHI'},
+    'OCH': {'FEM', 'MAL', 'OCH', 'CHI'},
+}
+# Work in progress, method and parameters may evolve
 def conversations(annotations: pd.DataFrame,
-                  interlocutors_1=('CHI',),
-                  interlocutors_2=('FEM', 'MAL', 'OCH'),
+                  interactions = INTERACTIONS,
                   max_interval=5000,
                   min_delay=0):
 
```

Comment:
We are shifting from two groups of speakers, each of the first group having the possibility of interacting with the members of the other group to a fully descriptive dictionary of interactions.
The first method was confusing and did not offer the ability to do any combination wanted.
The new method is giving a dictionary where each key is a speaker and contains the speakers it can 'answer' to in order for its vocalization to be considered a turn. The representation might still lack some clarity but it lets users define any combination.